README.md

```diff
@@ -1,137 +1,78 @@
-# MediaJel Engineering Coding Challenge
+# Installation
```

Comment:
For this PR that is getting merged into Master we want to keep the essence of the code challenge because it will be given to future applicants. These are great and easy to follow instructions - but should be integrated into the existing README along with rewording the challenge description to reflect your changes.

Reply:
I will take this into account. So i will add this to the README too.